Containerfile

```diff
@@ -1,6 +1,9 @@
 ### Create runtime image fase
 FROM docker.io/eclipse-temurin:17-jre-focal as runtime
 
+# Include OpenTelemetry javaagent
+ADD https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.32.0/opentelemetry-javaagent.jar .
```

Comment:
Is there maybe a nicer way to do this? Because Dependabot will not be able to update this dependency now. Not sure what the best practise is here..